config/initializers/cookies_serializer.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.action_dispatch.cookies_serializer = :json
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
I think this commit message answers why? The details are basically "I'm not sure why this file existed before, I don't think it applied, this should be better?

"906de238867e9806c7d894d16ccc23937bd8fcd5